lib/modules/datasource/postprocess-release.ts

```diff
@@ -49,6 +49,20 @@ export async function postprocessRelease(
     );
 
     if (result === 'reject') {
+      if (config.extractVersion) {
+        logger.debug(
```

Comment:
Trace maybe?

Reply:
Don't know, I think it's rare combination, so maybe it's good to have it at the debug level